sdjournal/journal.go

```diff
@@ -426,6 +435,33 @@ func NewJournalFromDir(path string) (j *Journal, err error) {
 	return j, nil
 }
 
+// NewJournalFromFiles returns a new Journal instance pointing to a journals residing
+// in a given files. The supplied path may be relative or absolute; if
+// relative, it will be converted to an absolute path before being opened.
```

Comment:
Where is this abs() applied? I didn't find it neither here nor mentioned in the `sd_journal_open` manpage.